peer_mentoring/posts/models.py

```diff
@@ -15,5 +16,7 @@ class Post(models.Model):
     author = models.ForeignKey(User, on_delete=models.CASCADE)
     title = models.CharField(max_length=150)
     content = models.TextField(blank=False, null=True)
-    comments = models.ManyToManyField(Comments)
+    comments = models.ManyToManyField(Comment)
```

Comment:
So a comment can be tied to multiple posts? Typical post-comment is 1-to-many

Reply:
Done